test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -60,4 +65,17 @@ public static Person generateTestPerson() {
             return null;
         }
     }
+    
+    public static void compareFiles(Path path1, Path path2) throws IOException {
```

Comment:
1. That's a valid point about using JUnit assert.  
2. I'm ok with using `Text` in the name to indicate text mode. `assertTextFilesEqual()` sounds good.